src/components/shared/Button/index.jsx

```diff
@@ -2,11 +2,7 @@ import React from "react";
 import { StyledButton } from "./styles";
 
 const Button = ({ clickEvent }) => {
-  return (
-    <StyledButton type="checkbox" onClick={clickEvent}>
-      Login
-    </StyledButton>
-  );
+  return <StyledButton onClick={clickEvent}>Login</StyledButton>;
```

Comment:
You created reusable component with fixed content `Login`. Get button content via props. See `children` prop